### PR TITLE
Update common/scripts/lib/_helpers.sh to list the machine ips before assigning admiral_ip

### PR DESCRIPTION
**Pull request by deepikasl**
https://github.com/Shippable/admiral/issues/1503
  
screenshot from 2018-01-05 16 55 22

### CODE REVIEW

File: common/scripts/lib/_helpers.sh [hunk 1]
```diff
@@ -303,6 +303,12 @@ __set_secret_key() {
   __process_success "Please enter the provided installer secret key."
   read response
   export SECRET_KEY="$response"
+
+  local get_ips=$(ip -f inet -4 addr | grep inet | grep -v 127.0.0.1 | awk '{print $2}' | awk -F "/" '{print $1}');
+
+  if [[ "$get_ips" ]]; then
```
Comment (ric03uec) [inline common/scripts/lib/_helpers.sh]: add a logger here in both if and else blocks to say whether ips are found or not. 
- if found, then something like `x ip addresses available for the host`
- if not, `no ip addresses avaiable`

File: common/scripts/lib/_helpers.sh [hunk 2]
```diff
@@ -313,20 +319,36 @@ _validate_ip() {
...
+
+  for (( i = 1 ; i < ${#machine_ips[@]} + 1 ; i++ )) do
+    __process_msg "$i - (${machine_ips[$i-1]})"
+  done
+
+  __process_success "Please pick one of the IP addresses from the above list by selecting the index (1, 2, etc) or enter a custom IP"
   read response
 
-  if [ "$response" != "D" ] && [ "$response" != "d" ]; then
+  if [[ $response =~ ^[0-9]+$ ]]; then
```
Comment (ric03uec) [inline common/scripts/lib/_helpers.sh]: add a logger after this, something like `1.1.1.1 was selected as admiral ip`

File: common/scripts/lib/_helpers.sh [hunk 3]
```diff
@@ -313,20 +319,36 @@ _validate_ip() {
...
+    if [ ${machine_ips[$response-1]} ]; then
+      export ADMIRAL_IP=${machine_ips[$response-1]}
```
Comment (ric03uec) [inline common/scripts/lib/_helpers.sh]: add logger afeter this, `admiral ip is set to ...`

Comment (ric03uec) [reply common/scripts/lib/_helpers.sh]: we want more verbosity here to make sure user knows exactly what is happening with each step they're taking. we can later hide some of the redundant stuff but the basic assumption should be that user should be shown everything

Comment (deepikasl) [reply common/scripts/lib/_helpers.sh]: ok, i will update it.

**Pull request merged**